README.adoc

```diff
@@ -331,6 +331,44 @@ spec:
 ...
 ----
 
+=== LDAP Configuration
+To configure authentication strategy to LDAP add the following configuration to the yaml file. If the auth strategy is LDAP. The application will not start without the basic LDAP settings like ldapHost, ldapPort, ldapBase and ldapBindDN.
+
+----
+auth:
+  strategy: ldap
+  ldap:
+    # AD IP host and port
+    ldap_host: example.com
+    ldap_port: 123
+    ldap_use_ssl: false
+    ldap_insecure_skip_verify: true
+    ldap_user_ilter: (cn=%s)
+    ldap_group_filter: (cn=%s)
+    ldap_attributes: ['memberOf', 'cn', 'mail']
```

Comment:
This `ldap_attributes` has been removed and replaced with individual ones for member, cn, and mail - these docs should reflect those changes.